laba3.cpp

```diff
@@ -0,0 +1,50 @@
+﻿#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+
+#define str_size 100 
+
+void main()
+{
+    char str[str_size];
+    int alp, digit, splch, i;
+    alp = digit = splch = i = 0;
```

Comment:
what is splch?

Reply:
symbols, characters like , .(gap), I name them special characters